library/Icingadb/Widget/Detail/MultiselectQuickActions.php

```diff
@@ -27,10 +26,45 @@ class MultiselectQuickActions extends BaseHtmlElement
 
     protected $defaultAttributes = ['class' => 'quick-actions'];
 
-    public function __construct($type, $summary)
+    /** @var ?string Url path for the {@link getLink()} method */
+    protected $urlPath;
+
+    /** @var bool Whether the rendering process-check-result button should be ignored */
+    protected $ignoreProcessCheckResultButton;
+
+    public function __construct($type, $summary, bool $ignoreProcessCheckResultButton = false)
     {
         $this->summary = $summary;
         $this->type = $type;
+        $this->ignoreProcessCheckResultButton = $ignoreProcessCheckResultButton;
+    }
+
+    /**
+     * Get the URL path for the {@link getLink()} method
+     *
+     * @return string
+     */
+    public function getUrlPath(): string
+    {
+        if ($this->urlPath === null) {
+            $this->urlPath = "icingadb/{$this->type}s";
+        }
+
+        return $this->urlPath;
+    }
+
+    /**
+     * Set the URL path for the {@link getLink()} method (omits the trailing slash)
+     *
+     * @param string $urlPath
+     *
+     * @return $this
+     */
+    public function setUrlPath(string $urlPath): self
```

Comment:
hmm, what do you think about only allowing to adjust whether a singular or plural version of the type should be used?